lib/modules/manager/pep621/processors/uv.ts

```diff
@@ -32,6 +33,11 @@ export class UvProcessor implements PyProjectProcessor {
       ),
     );
 
+    const registryUrls = extractRegistryUrls(uv);
+    for (const dep of deps) {
+      dep.registryUrls = [...registryUrls];
```

Comment:
Won't this now always set registry URLs? Isn't it better to leave it blank if none are explicitly set?

Reply:
Yep we can, I was hesitant to do it because PDM also always set the registry URLs AFAIK, even if the datasource itself also has the default, but I also prefer to only override if necessary in the manager itself, so updated that, thanks.

Reply:
I'll need to check here with @secustor on whether that's intentional or not. Generally speaking we prefer that managers don't set registryUrls unless they are specified, otherwise we should let the datasource defaults take precedence

Reply:
I have set this in PDM on purpose as in case of the manager all registryUrls set in their tool context also applies to "standard" dependencies of `pyproject.toml` 

`registryUrls` should be only set in case of PDM a `tool.pdm.source` table exists
```
[[tool.pdm.source]]
name = "private"
url = "https://private.pypi.org/simple"
verify_ssl = true
```